### ISSUE TITLE
Loadgen should have external hipstershop ip not editable

### ISSUE THREAD

**Issue by simonz130** [GoogleCloudPlatform/cloud-ops-sandbox#561]
We want to prevent users from using loadgen to send traffic to resources that are not part of what was provisioned in the Sandbox.

In addition, it would be nice to be able to deploy the Sandbox with or without the loadgen.

**Comment by Daniel-Sanche** [comment]
Should we disable the entire webUI? I imagine we shouldn't give users direct control over the spawn size either if they're not paying for the resources used

**Comment by Daniel-Sanche** [comment]
I solved the second part of the request here: https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/pull/584

Let me know what you think about the web UI

**Comment by minherz** [comment]
I do not think that the first part is doable since LoadGen can be configured to send traffic anywhere.
